txdbus/authentication.py

```diff
@@ -540,6 +547,7 @@ def beginAuthentication(self, protocol):
         
 
     def handleAuthMessage(self, line):
+        line = six.b(line)
```

Comment:
Agreed. I just sometimes hesitate if our public API should accept strings [or not].